akvo/rsr/spa/app/modules/results-admin/TobeReported.jsx

```diff
@@ -147,96 +155,155 @@ const TobeReported = ({
-                  <br />
-                  {((!isEmpty(item?.indicator?.description.trim())) && item?.indicator?.description?.trim().length > 5) && (
-                    <details>
-                      <summary>{t('Description')}</summary>
-                      <p className="desc hide-for-mobile">{mdOutput(mdParse(item?.indicator?.description))}</p>
-                    </details>
-                  )}
-                </Col>
-                <Col lg={2} md={2} sm={24} xs={24} className="action">
-                  {
-                    (activeKey === iKey)
-                      ? (
-                        <div className="action-close">
-                          <Button onClick={handleCancel}>
-                            <Icon type="close" />
-                            <span className="action-text">Close</span>
-                          </Button>
-                        </div>
-                      )
-                      : (
-                        <Button
-                          type="link"
-                          onClick={() => {
-                            if (errors.length) {
-                              setErrors([])
-                            }
-                            handleOnEdit(item)
-                            setActiveKey(iKey)
-                          }}
-                          block
-                        >
-                          <SVGInline svg={editButton} className="edit-button" />
-                          <span className="action-text">Edit Value</span>
+    <>
+      <AuditTrailModal {...openHistory} onClose={handleOnCloseModal} />
+      <List
+        grid={{ column: 1 }}
+        itemLayout="vertical"
+        className="tobe-reported"
+        dataSource={updates}
+        renderItem={(item, ix) => {
+          const iKey = item?.id || `${item?.indicator?.id}0${ix}`
+          const allSubmissions = resultRdr
+            ?.filter((r) => r.id === item.result?.id)
+            ?.flatMap((r) => r.indicators)
+            ?.filter((i) => i.id === item.indicator?.id)
+            ?.flatMap((i) => i.periods)
+            ?.filter((p) => (
+              p.id === item.period?.id &&
+              p.updates.length &&
+              !p.locked
+            ))
+            ?.flatMap((p) => p.updates)
+          const updateClass = (!item.status && allSubmissions.length) ? ACTIVE_PERIOD : kebabCase(item?.statusDisplay)
+          return (
+            <List.Item className="tobe-reported-item">
+              <Card className={classNames(updateClass, { active: (activeKey === iKey) })}>
+                <Row type="flex" justify="space-between" align="top">
+                  <Col lg={22} md={22} sm={24} xs={24}>
+                    {isEmpty(period) && (
+                      <div className="period-caption">
+                        {moment(item?.period?.periodStart, 'DD/MM/YYYY').format('DD MMM YYYY')} - {moment(item?.period?.periodEnd, 'DD/MM/YYYY').format('DD MMM YYYY')}
```

Comment:
It would be nice to create variable and assign values and render variable. If later on we convert it to component it would be easier to just pass the variable instead